crates/bench-macro/src/lib.rs

```diff
@@ -0,0 +1,86 @@
+//! See the README for `wasm-bindgen-test` for a bit more info about what's
```

Comment:
@aboodman, noting so I don't forget: ~85% of this file comes from https://github.com/rustwasm/wasm-bindgen/blob/master/crates/test-macro/src/lib.rs, which is dual Apache/MIT licensed. Want to make sure we correctly note that and honor the license when committing.

Reply:
One thing I have seen done is to add a note to that effect at the top and then paste in the appropriate license. Does that make sense to you?

Reply:
SG, coming up.